snappiershot/serializers/json.py

```diff
@@ -85,6 +86,7 @@ def encode_numeric(value: Number) -> JsonType:
 
         This will do nothing to naturally serializable types (bool, int, float)
           but will perform custom encoding for non-supported types (complex).
+        This will cast Decimal values to float values.
```

Comment:
This conversion is lossy. I think [the `Decimal.from_float` docs](https://docs.python.org/3.9/library/decimal.html#decimal.Decimal.from_float) summarize this pretty well. We should maybe the `Decimal.as_tuple` method which seems to handle many edge cases for use, such as `-0`